_includes/integrations/expressjs-quick-start.md

```diff
@@ -0,0 +1,140 @@
+This quick start guide walks through creating a [basic Express JS application](https://expressjs.com/en/starter/generator.html) containing a {{site.productname}} editor based on our [Basic example]({{site.baseurl}}/demo/basic-example/).
+
+> **NOTE**: {{site.cloudname}} users should use the [{{site.cloudname}} quick start guide]({{site.baseurl}}/quick-start/).
+
+### Prerequisites
+
+This procedure requires:
+
+* [Node.js (and npm)](https://nodejs.org/).
+
+### Procedure
+
+1. On a command line or command prompt, create a basic Express JS application:
+
+    ```sh
+    $ npx express-generator --git --no-view my_{{site.prodnamecode}}_app
+    ```
+
+1. Change into the newly created directory.
+
+    ```sh
+    $ cd my_{{site.prodnamecode}}_app
+    ```
+
+1. Add the `tinymce` package to your `package.json` with `--save`.
+
+    ```sh
+    $ npm install --save tinymce
+    ```
+
+1. Download and install the {{site.productname}} and Express JS Node modules (and their dependencies):
+
+    ```sh
+    $ npm install
+    ```
+
+1. Using a text editor, open `/path/to/my_{{site.prodnamecode}}_app/app.js` and add the following route:
+
+    ```jsx
```

Comment:
This shouldn't be `jsx`, it's just plain `js`.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1625/commits/1f077fff34ce2fbd9db25ca8b5bfee299e9e881c